haskell-open-source.md

```diff
@@ -64,3 +64,27 @@ To determine what changes are "breaking", use the following:
 ![](./pvp-chart.png)
 
 With `A=EPOCH`, `B=MAJOR`, and `C=MINOR`.
+
+## Release
+
+All (interested) Frecklers should be in the Maintainers group of our Hackage
+libraries, with permission to upload new versions.
+
+To release a new version:
+
+1. Update the package version
+1. Add a `CHANGELOG` entry
```

Comment:
2. 3. 4. ?

Reply:
Wtf is that styling GH, Jesus. Was asking if you meant to enumerate beyond the number 1?